proto/panacea/datapool/v2/pool.proto

```diff
@@ -19,6 +19,7 @@ message Pool {
   string                   curator = 8;
   cosmos.base.v1beta1.Coin deposit = 9 [(gogoproto.nullable) = false]; // deposit
   string                   nft_contract_addr = 10;                     // NFT contract address of pool
+  bool                     is_paid_deposit = 11;
```

Comment:
Does this mean that the curator was paid the deposit to the pool? Or, that the deposit was returned to the curator?

If the latter, it would be clearer to rename this to `was_deposit_returned`.

Reply:
Yes, you're right.
I will change this name soon.

Reply:
I fixed it.
[change is_paid_deposit to was_deposit_returned](https://github.com/medibloc/panacea-core/pull/318/commits/ecf72ead29d4c0a2cb027b147c8648b03b924d56)